invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js

```diff
@@ -61,7 +61,10 @@ export class ManagerMembersResultItem extends Component {
     const { config } = this.props;
     const { result } = this.state;
     const { api } = this.context;
-
+    const membershipRelativeTimestamp = timestampToRelativeTime(result.created);
+    const memberVisibility = result.visible
+      ? i18next.t(config.visibility[0].title)
```

Comment:
this will not work with translations as passing a variable is not extracting the dynamic value on runtime... why do we need this change?